tests/migrations/test_executor.py

```diff
@@ -39,10 +40,17 @@ def test_run(self):
         self.assertTableNotExists("migrations_author")
         self.assertTableNotExists("migrations_book")
         # Alright, let's try running it
-        executor.migrate([("migrations", "0002_second")])
+        with patch_logger('django.db.migrations', 'info') as logs:
+            executor.migrate([("migrations", "0002_second")])
         # Are the tables there now?
         self.assertTableExists("migrations_author")
         self.assertTableExists("migrations_book")
+        self.assertEqual(logs, [
+            'Applied migration migrations.0001_initial on database default',
+            'Marked migration migrations.0001_initial as applied',
+            'Applied migration migrations.0002_second on database default',
+            'Marked migration migrations.0002_second as applied',
+        ])
```

Comment:
I'd use tests that are slightly less sensitive to the exact phrasing of the log messages -- counting the log messages, making sure the right migration names are where I'd expect them, but not compare the entire log line. But maybe that's just me.